common/app/views/fragments/contentMeta.scala.html

```diff
@@ -41,15 +41,16 @@
             }
             @fragments.meta.contactAuthor(item.tags, amp)
         }
+
+        @if(!(item.trail.shouldHidePublicationDate || item.content.isGallery)) {
```

Comment:
This change will apply to everyone, are you sure about that or is it Garnett-specific?